pootle/apps/pootle_store/fields.py

```diff
@@ -187,6 +187,7 @@ def _update_store_cache(self):
                 from pootle_store.filetypes import factory_classes
 
                 store_obj = factory.getobject(self.path,
+                                              localfiletype=self.instance.translation_project.project.localfiletype,
```

Comment:
Thanks. This was really helpful. Next time please warn about dependencies in the PR. I just added this one myself.